ros_ign_image/test/test_utils.h

```diff
@@ -137,7 +137,7 @@ namespace testing
   }
 
   //////////////////////////////////////////////////
-  /// Ignition::msgs test utils
+  /// Gazebo::msgs test utils
```

Comment:
Humm either

```suggestion
  /// Gazebo msgs test utils
```

or

```suggestion
  /// gz::msgs test utils
```